walletrpc/compact_formats.proto

```diff
@@ -28,7 +28,7 @@ message CompactBlock {
     bytes hash = 3;                  // the ID (hash) of this block, same as in block explorers
     bytes prevHash = 4;              // the ID (hash) of this block's predecessor
     uint32 time = 5;                 // Unix epoch time when the block was mined
-    bytes header = 6;                // (hash, prevHash, and time) OR (full header)
+    bytes header = 6;                // full header (as returned by the getblock RPC)
```

Comment:
I disagree that this is a minor clarification. The `OR` should have been moved into the documentation of `CompactBlock`, because the intention was that either `CompactBlock.header` is set, or `CompactBlock.{hash, prevHash, time}` are set.